### PR TITLE
Allow use of detached node, as well as text or html for `content`

### PR DESCRIPTION
**Pull request by rosskevin**
@see #27 - Use with detached (fabricated) DOM node fails

I changed option `allowHtml` to `allow` with options `text` (default) | `html` | `node` - supports adding dynamic/detached nodes with the #parse method and all it's goodness.

See #27 for the codepens.

### CODE REVIEW

**Comment by rosskevin** [comment]
BTW - I didn't commit build or re-run docs just so it cleanly showed changes.  I can run these in the future if you would like.

File: src/popper.js [hunk 1]
```diff
@@ -87,8 +87,8 @@
      * @param {Array} [popper.classNames=['popper']] Array of classes to apply to the generated popper.
      * @param {Array} [popper.attributes] Array of attributes to apply, specify `attr:value` to assign a value to it.
      * @param {HTMLElement|String} [popper.parent=window.document.body] The parent element, given as HTMLElement or as query string.
-     * @param {String} [popper.content=''] The content of the popper, it can be text or HTML, in case of HTML, enable `allowHtml`.
-     * @param {Boolean} [popper.allowHtml=false] If set to true, the `content` will be parsed as HTML.
+     * @param {String} [popper.content=''] The content of the popper, it can be text, html, or node; if it is not text, set `allow` to `html` or `node`.
+     * @param {String} [popper.allow=text] If `html`, the `content` will be parsed as HTML. If `node`, it will be appended as-is.
```
Comment (FezVrasta) [inline src/popper.js]: This should be `popper.allow='text'`

**Comment by FezVrasta** [comment]
I would rename `allow` to `contentType`

**Comment by FezVrasta** [comment]
About the build and docs tasks, it's okay to not commit them. I will run them when I'll release a new version.

**Comment by FezVrasta** [comment]
By the way, I don't feel comfortable in merging this PR until #25 is fixed.

I want to be sure that the initial placement of "injected nodes" works properly.

**Comment by rosskevin** [comment]
Update to `contentType`, I think that makes more sense.

**Comment by FezVrasta** [comment]
@rosskevin now that #25 is fixed I'd like to continue with this PR.

May you remove the build result from this PR and fix the conflicts? Thanks.

**Comment by rosskevin** [comment]
Ok, PR is all set. 

It contains the master's build artifacts so my test case won't work until it is pointed at a new build.

**Pull request merged**